OpenSim/Simulation/SimulationUtilities.h

```diff
@@ -103,9 +103,70 @@ inline SimTK::State simulate(Model& model,
 
     return state;
 }
-
 /// @}
 
+/** This function can be used to upgrade MOT files generated with versions
+    3.3 and earlier in which some data columns are associated with coordinates
+    that were incorrectly marked as Rotational (rather than Coupled). Specific
+    instances of the issue are the patella coordinate in the Rajagopal 2015 and
+    leg6dof9musc models. In these cases, the patella will visualize incorrectly
+    in the GUI when replaying the kinematics from the MOT file, and static 
+    optimization results.
+    Moving from OpenSim 3.3 (and earlier) to OpenSim 4.0, MotionTypes for
+    Coordinates are now determined strictly by the coordinates' owning Joint.
```

Comment:
Should the second "coordinates'" be capitalized?? I guess this depends on if you meant the class or coordinates more generically.